frontend/src/components/ViewActivities/ErrorComponents.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+
+import Button from 'react-bootstrap/Button';
+
+import { VIEW_TRIPS } from '../../constants/routes';
+
+/**
+ * A simple Button component that redirects the user back to the VIEW_TRIPS page
+ * that is rendered with each of this file's error components.
+ */
+const GoToTripsButton = () => {
+  const history = useHistory();
+
+  function goToTrips() {
+    history.push(VIEW_TRIPS);
+  }
+
+  return (
+    <Button type='button' onClick={goToTrips} variant='primary' size='lg'>
+      Go Back to Your Trips
+    </Button>
+  );
+}
```

Comment:
Does this component fall under "ErrorComponents"? If you think this is the best place to put it then that is totally fine.

Reply:
I've decided to put these error components into their own separate directory `components/Errors`. This should help with making this cleaner and more general for uses beyond the ViewActivities component.